monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -75,77 +28,55 @@ class HTTPIslandAPIClient(IIslandAPIClient):
     def __init__(
         self,
         agent_event_serializer_registry: AgentEventSerializerRegistry,
-        retries=RETRIES,
     ):
         self._agent_event_serializer_registry = agent_event_serializer_registry
-        self._session = requests.Session()
-        retry_config = Retry(retries)
-        self._session.mount("https://", HTTPAdapter(max_retries=retry_config))
+        self.request_facade = HTTPRequestsFacade("")
 
     @handle_island_errors
     def connect(
```

Comment:
Connect still has temporal coupling, but I'll leave it to another boy scout to fix.